setup.py

```diff
@@ -98,6 +98,12 @@ def run(self):
               'Operating System :: MacOS',
           ],
           platforms='any',
+          install_requires=[
+              'numpy >=1.14',
+              'NEURON >=7.7',
+              'matplotlib',
+              'scipy'
+          ],
```

Comment:
@cjayb what are your thoughts here? I just realized that `mne` is also technically a dependency (although a soft one) if we want to run the TFR functions. I want folks to be able to run the examples right after doing the install without having any errors ... could add MNE to the install_requires or maybe to extra_requires along with joblibs. But hesitating a bit to be honest.

Reply:
I imagine the dependency will become harder going forward, although right now it does seem an overkill. mne is beginning to be weighty in its own right, but given that it can be pip-installed, perhaps that's acceptable?

I think I'm leaning mildly towards including it as an explicit dependency. Most non-experts wouldn't blink about running an extra pip-install of a package they don't understand the function of. So let's just keep it simple and install it for them. It's not like our packaging is complicated.